stylesheets/_index.scss

```diff
@@ -24,7 +24,7 @@
   .content {
     background-color: $grey_l;
     flex-grow: 1;
-    overflow-y: auto;
+    overflow-y: overlay;
```

Comment:
This is non-standard, right? That's fine as long as it works, but it's something to keep in mind in the long term since chromium could always remove support for it.